src/hooks/core/chat/useSystemMessage.ts

```diff
@@ -156,16 +163,19 @@ const useSystemMessage = () => {
       }
 
       if (__isMessageForOtherUser(sortedMessages[0])) {
+        if (withLog) console.log('checkpoint5');
         firstMessage = checkSystemMessageOnlyForOtherSystemUser(sortedMessages[0]);
         break;
       }
 
       if (__isSystemMessage(sortedMessages[0])) {
+        if (withLog) console.log('checkpoint6');
         firstMessage = checkSystemMessage(sortedMessages[0]);
         break;
       }
 
       if (checkFirstMessage?.text?.toLocaleLowerCase()?.includes('started following')) {
+        if (withLog) console.log('checkpoint7');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, avoid using `console.log` in production code. Use a proper logging library instead.


<!-- This is an auto-generated comment by OSS CodeRabbit -->